src/__tests__/github.mapping.test.ts

```diff
@@ -0,0 +1,99 @@
+import { mapProject, mapProjects } from '../mapper/github.mapping';
+
+describe('mapProject', () => {
+  it('should map a repository object to a project object', () => {
+    const repository = {
+      name: 'test-repo',
+      description: 'This is a test repository.',
+      url: 'https://github.com/test/test-repo',
+      language: 'Python',
+    };
+    const languages = ['Python', 'JavaScript'];
+    const contributors = [
+      { login: 'johndoe', avatarUrl: 'https://example.com/avatar.png' },
+    ];
+
+    const project = mapProject(repository, languages, contributors);
+
+    expect(project).toEqual({
+      name: 'test-repo',
+      description: 'This is a test repository.',
+      url: 'https://github.com/test/test-repo',
+      language: 'Python',
+      contributors: [
+        { name: 'johndoe', avatarUrl: 'https://example.com/avatar.png' },
+      ],
+      languages: ['Python', 'JavaScript'],
+    });
+  });
+});
+
+describe('mapProjects', () => {
+  it('should map an array of repository objects to an array of project objects', () => {
+    const repositories = [
+      {
+        name: 'test-repo1',
+        description: 'This is the first test repository.',
+        url: 'https://github.com/test/test-repo1',
+        language: 'Python',
+      },
+      {
+        name: 'test-repo2',
+        description: 'This is the second test repository.',
+        url: 'https://github.com/test/test-repo2',
+        language: 'JavaScript',
+      },
+    ];
+    const repositoriesLanguages = {
+      'test-repo1': ['Python', 'JavaScript'],
+      'test-repo2': ['JavaScript', 'CSS', 'HTML'],
+    };
+    const repositoriesContributors = {
+      'test-repo1': [
+        { login: 'janedoe', avatarUrl: 'https://example.com/avatar2.png' },
+      ],
+      'test-repo2': [
+        {
+          login: 'johndoe',
+          avatarUrl: 'https://example.com/johndoe_avatar.png',
+        },
+      ],
+    };
+    const repositoriesLanguagesArray = Object.values(repositoriesLanguages);
```

Comment:
why do you need to do this?

This might be why your test are failing.

Reply:
Yeah, I removed that part. The problem was that JSlint showed an error. I tried to use object.values to fix it.